README.md

```diff
@@ -17,4 +17,14 @@ Current version: [![Current Version](https://img.shields.io/npm/v/catbox-memory.
   to an object using `JSON.parse()` on retrieval. By setting this option to `true`,
   `Buffer` data can be stored alongside the stringified data. `Buffer`s are not 
   stringified, and are copied before storage to prevent the value from changing while
-  in the cache. Defaults to `false`.
\ No newline at end of file
+  in the cache. Defaults to `false`.
+  
+### What to Use catbox-memory for and When to Use It
+
+**catbox-memory** is a memory caching module for **catbox**.
+
+**catbox** is a multi-strategy key-value object store. It allows a developer to more easily manage server side caching and storage rules.
+
+However, **catbox** does not include an external memory cache. That's where **catbox-memory** comes in. **catbox-memory** can be used to cache objects to memory.
+
+**catbox-memory** can be particularly helpful if you're trying to increase the efficiency of a legacy API system. The module can be used to cache the results of several API/database calls to memory, reducing bandwidth and server load and increasing the speed of an application.
```

Comment:
I don't think this statement really belongs in the README. catbox-memory is an in memory cache for catbox, and people can use it for whatever suits them. If you have a good success story with caching API calls, maybe turn that into a blog post with more detail.